### PR TITLE
Updated G Web examples

### PR DESCRIPTION
**Pull request by abahetik**
# Justification
Replacing the LabVIEW Web Module webvi-examples with G Web Development Software examples. I have updated the readmes accordingly.

# Implementation
1. Replaced the LV Web module examples with the new G Web examples.
2. Retained the Readme files from the previous examples and updated few contents like:
    1. The Web Module and LabVIEW NXG references.
    2. Replaced documentation link with the new G Web links
    3. Updated few images images and gifs with G Web images.
    4. Removed System Designer workflows.
    5. Updated instructions based on the new expected directory names.

# Remaining works
1. Updating the forum details ( will be done after G Web forums is created)
2. Building the new gwebprojs. ( will do this after the release)
3. Updating the github.io urls which will host the new projects

### CODE REVIEW

**Review by rajsite — CHANGES_REQUESTED** [review]

File: CallLabVIEWWebService/Readme.md [hunk 1]
```diff
@@ -20,21 +20,23 @@ Use LabVIEW to create web services that do what WebVIs can't do, such as:
 
 ## Dependencies
 
-- LabVIEW NXG Web Module
+- G Web Development Software
 - LabVIEW 2016 or later
```
[the remark this answers is not in the document]
Comment (abahetik) [reply CallLabVIEWWebService/Readme.md]: Corrected and pushed new WebServices folder. The .lvproject was fixed using LabVIEW 2018. Have updated the readme accordingly

**Comment by rajsite** [comment]
@abahetik I'm not clear if the PR is ready to review and merge in the gweb branch. Will the unresolved comments be handled in a follow-up PR or am I keeping this PR open waiting for those changes? Similar to ASW please comment and / or resolve comments and reset reviewers when ready 👍

**Comment by abahetik** [comment]
Hey Milan, I have implemented the recommended readme changes in the PR. There are a few things left that I will address in the next PR.
1. The banner color change - I have the projects ready but there are two reasons why I am not doing it in this PR. First, There are a lot of files affected. Second, The gifs need to be recorded and since one of the [WebVI examples](https://ni.github.io/webvi-examples/EmbedContentIntoWebVI/Builds/WebApp_Web%20Server/) is broken, it will look odd to not update this example or record the gif with a broken WebCam.
2. For `WebApp.lvdist for Call3rdPartyWebService is broken`, when I rebuilt the lvdist it started working so I will push it in the next PR where I will push all the Build repos for the examples(post release).
3. The Forum updates, for this comment I will push the changes once the G Web fourm is live.

I m resolving the comments for the above. Please review the PR and allow the merge. Let me know if there are any other updates required.

**Comment by rajsite** [comment]
@abahetik I thought it would be quicker to push some changes directly. Summary of my updates:
- The Call3rdpartyWebService lvdist still failed to build when opened and pressing build. This was because none of the previous metadata was copied over. I updated the metadata to match the previous lvdist.
- The project cache for the SystemLink example was updated to no longer open on index.gviweb by default. Fixed the project cache.
- The demo.tdms_index cache file was checked in for some reason. Deleted it.
- Updated additional stale NXG references in comments for the LabVIEW projects and GWeb projects
- Reverted the hosting update. The PR comment has stated to add a fourth hosting option (that was listed first). Instead the first option was changed. It's not clear if that was intentional as that was resolved without a comment. Regardless I think we need a more comprehensive update to our hosting discussion in the examples as a lot has improved since the initial releases when they were written.
- Ran the spelling and markdown linters and updated files

**Pull request merged**